src/components/Modals/Send/AddressInput/AddressInput.tsx

```diff
@@ -43,11 +50,17 @@ export const AddressInput = ({ rules, placeholder, enableQr = false }: AddressIn
         data-test='send-address-input'
         data-1p-ignore
         // Because the InputRightElement is hover the input, we need to let this space free
-        pe={10}
+        pe={pe}
         isInvalid={!isValid}
+        // This is already a `useCallback()`
+        // eslint-disable-next-line react-memo/require-usememo
+        sx={{
+          fontSize: value && value.length > 42 ? '12px' : '13.5px',
```

Comment:
0.5px smaller than the regular `14px` sm to accomodate for regular sends looking off
<img width="490" alt="image" src="https://github.com/user-attachments/assets/43939bbe-8d61-4edc-b487-515706f773b9">

Reply:
The only thing that could be weird is when typing manually, but it should never happen, and if it happens it's weird anyway, so it looks acceptable to have a weird size changing when typing 👀